python/create_mask.py

```diff
@@ -0,0 +1,73 @@
+from datetime import datetime
+import glob
+import sys
+import mahotas
+import numpy as np
+import matplotlib.pyplot as plt
+
+if len(sys.argv) > 1:
+    path = sys.argv[1]
+else:
+    path=''
```

Comment:
I would write:
```python
path = ''
if len(sys.argv) > 1:
    path = sys.argv[1]
```

Though... are you sure your first argument is going to be a path?? this could break badly!

Reply:
Changed to your format and added some structure to verify path is indeed an existing path location